docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -282,3 +282,64 @@ DSA keys look almost identical but begin with ``ssh-dss`` rather than
 
     :raises cryptography.exceptions.UnsupportedAlgorithm: If the serialized
         key is of a type that is not supported.
+
+Serialization Formats
+~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: Format
+
+    .. versionadded:: 0.8
+
+    An enumeration for key formats.
```

Comment:
I feel like a little bit denser linking might be helpful for someone reading this documentation.  For example, a link here back to `RSAPrivateKeyWithSerialization` to say that that's where this enumeration is used would be helpful.